performance/SwiftUIPerformanceQuickstart/PerformanceExample/PerformanceExample.xcodeproj/project.pbxproj

```diff
@@ -16,9 +16,9 @@
 		41BFA1EC26B1D3730013EB90 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 4102055826A631F5004B24B3 /* Assets.xcassets */; };
 		41BFA1F026B1D8B80013EB90 /* Process.swift in Sources */ = {isa = PBXBuildFile; fileRef = 41BFA1EF26B1D8B80013EB90 /* Process.swift */; };
 		41BFA1F326B1D8B80013EB90 /* Process.swift in Sources */ = {isa = PBXBuildFile; fileRef = 41BFA1EF26B1D8B80013EB90 /* Process.swift */; };
-		5A81EBDA3528749141B3873C /* Pods_PerformanceExample__iOS_.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = F4171CE0A02526CAA1C76922 /* Pods_PerformanceExample__iOS_.framework */; };
-		62EF508442D77FC8E18D3D05 /* Pods_PerformanceExample__tvOS_.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 5881298C295F336EB049ADBE /* Pods_PerformanceExample__tvOS_.framework */; };
-		E0289F7ACA086958C3091522 /* Pods_Tests_iOS.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = A7DBF8AF48A3164ECEA5CD0E /* Pods_Tests_iOS.framework */; };
+		84EF5C77BCF64159DC1554B3 /* Pods_Tests_iOS.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 2766988506B7F4F19C97E851 /* Pods_Tests_iOS.framework */; };
+		92569034A655AF847A57DC68 /* Pods_PerformanceExample__tvOS_.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = DA26F920681B148132EA1D2D /* Pods_PerformanceExample__tvOS_.framework */; };
+		E39F69BE4EC5DA0B8BCB2B3B /* Pods_PerformanceExample__iOS_.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 33530B44769A29407676BEA8 /* Pods_PerformanceExample__iOS_.framework */; };
```

Comment:
Since I will be squashing and merging, any intermediary changes in these commits will not show up in the diff (since this is a new directory all files committed will be wholly new).

Reply:
`pod deintegrate` still needs to be run to avoid adding Pods references to the repo.

Reply:
Done ✅